docs/source/cdk/cdk-deployment.rst

```diff
@@ -0,0 +1,30 @@
+CDK deployment
+==============
+Deploy
+~~~~~~
+You will need to make a **copy** of app_template_dev.py file with a different name (app__dev.py) and keep a copy of
+it locally so that it will not be committed. In your own copy there are two important configuration
+items which you can alter:
+
+#. AWS_PROFILE ()
+#. Your initials when deploying to an AWS account with multiple users ()
+
+To deploy, first set the appropriate environment variables::
+
+        export AWS_PROFILE=<profile>
+You'll then need to synthesize the CDK code with the command::
+
+        cdk synth --app "python app_template_dev.py"
```

Comment:
```suggestion
        cdk synth --app "python app_<initials>_dev.py"
```
I'm always torn on whether to put something runnable in, or not. I don't think we want people accidentally modifying `app_template_dev.py` here though, so would slightly lean towards putting the placeholder in instead?

Reply:
Agree